### PR TITLE
Adding UK provisioning configuration

### PR DESCRIPTION
**Pull request by lin-d-hop**

### CODE REVIEW

File: inventory/group_vars/all.yml [hunk 1]
```diff
@@ -15,15 +15,15 @@ ssh_pubkey: "{{ lookup('file', '~/.ssh/{{ ssh_pubkey_name }}') }}"
...
+git_version:  v1.10.1
 
 l10n_git_version: HEAD
 
 
 #----------------------------------------------------------------------
 # Default values (should be overridden in each server config)
 rails_env: staging
-admin_email: "admin@{{ domain }}"
+admin_email: "hello@{{ domain }}"
```
Comment (enricostano) [inline inventory/group_vars/all.yml]: This change would affect all the instances, that's why is in `all.yml` file. Any specific reason to change it? You should be able to override it in your specific `uk.yml` file. Makes sense? Can you please try that?

File: inventory/group_vars/uk.yml [hunk 2]
```diff
@@ -15,4 +15,12 @@ timezone: London
 developer_email: dev.ofnuk@gmail.com
 
 users_sysadmin:
-  - pmackay
+  - lindhop
+  - enricostn
```
Comment (enricostano) [inline inventory/group_vars/uk.yml]: I see that you add Matt's SSH public key in `/files` but you don't add him in this list.

File: inventory/group_vars/uk.yml [hunk 3]
```diff
@@ -15,4 +15,12 @@ timezone: London
...
+
+s3_backups_bucket: none
+s3_access_key: 666
+s3_secret: 666
+
+stripe_client_id: 666
+stripe_instance_secret_key: 666
```
Comment (elf-pavlik) [inline inventory/group_vars/uk.yml]: you will not need all above once #155 gets merged

**Comment by elf-pavlik** [comment]
@lin-d-hop would you still like this to have this PR merged?

**Comment by Matt-Yorkley** [comment]
This can be closed now, we tidied up all the UK config stuff in #171.

**Pull request closed without merging**